app/models/experiment.rb

```diff
@@ -98,7 +98,7 @@ def completed?
 
   def get_statistics
     all = simulation_runs.count
-    sent = simulation_runs.where(to_sent: false, is_done: false).count
+    sent = simulation_runs.where(is_done: false, to_sent: false).count
```

Comment:
Z tego co kojarzę, to hashe w Rubym nie trzymają kolejności w żaden sposób. Czy w takim razie zmiana kolejności cokolwiek zmienia?